### ISSUE TITLE
bumblebeed error: "Unloading nvidia driver timed out."

### ISSUE THREAD

**Issue by jethrogb** [Bumblebee-Project/Bumblebee#681]
Recently I've been getting issues where bumblebee doesn't turn the dGPU off properly. This is usually accompanied by error messages such as these in syslog:
```
bumblebeed[5664]: Unloading nvidia driver timed out.
```
Trying to use the dGPU in this state results in errors:
```
bumblebeed[5664]: [XORG] (EE) NVIDIA(GPU-0): Failed to initialize the NVIDIA GPU at PCI:1:0:0.  Please
bumblebeed[5664]: [XORG] (EE) NVIDIA(GPU-0):     check your system's kernel log for additional error
bumblebeed[5664]: [XORG] (EE) NVIDIA(GPU-0):     messages and refer to Chapter 8: Common Problems in the
bumblebeed[5664]: [XORG] (EE) NVIDIA(GPU-0):     README for additional information.
bumblebeed[5664]: [XORG] (EE) NVIDIA(GPU-0): Failed to initialize the NVIDIA graphics device!
bumblebeed[5664]: [XORG] (EE) NVIDIA(0): Failing initialization of X screen 0
bumblebeed[5664]: [XORG] (EE) Screen(s) found, but none have a usable configuration.
bumblebeed[5664]: [XORG] (EE) 
bumblebeed[5664]: [XORG] (EE) no screens found(EE) 
bumblebeed[5664]: [XORG] (EE) 
bumblebeed[5664]: [XORG] (EE) Please also check the log file at "/var/log/Xorg.8.log" for additional information.
bumblebeed[5664]: [XORG] (EE) 
bumblebeed[5664]: [XORG] (EE) Server terminated with error (1). Closing log file.
bumblebeed[5664]: X did not start properly
```
Yet when I do:
```
sudo rmmod nvidia
sudo restart bumblebeed
```
Everything works properly again.

I mostly use the intel-virtual-output bumblebee client.

I'm on a Lenovo Thinkpad T420s with hardware:
```
00:02.0 VGA compatible controller: Intel Corporation 2nd Generation Core Processor Family Integrated Graphics Controller (rev 09)
01:00.0 VGA compatible controller: NVIDIA Corporation GF119M [NVS 4200M] (rev a1)
```

I use Ubuntu 14.04 64-bit. Here are some package versions I use:
```
bumblebee                       3.2.1-91~trustyppa1
bbswitch-dkms                   0.8-2~trustyppa1
linux-image-3.13.0-52-generic   3.13.0-52.86
nvidia-346                      346.87-0ubuntu0~xedgers14.04.1
xserver-xorg                    1:7.7+1ubuntu8.1
xserver-xorg-core               2:1.15.1-0ubuntu2.7
xserver-xorg-video-intel        2:2.99.916+git20140911.79736944-0ubuntu0sarvatt~trusty
```

**Comment by Vincent-C** [comment]
This is likely caused by https://github.com/Bumblebee-Project/bumblebee-ppa/commit/2d3983e3d70db409c930ae5af2b3d538877854cf (included in the last PPA update for bumblebee) and will be fixed shortly in 3.2.1-92 which I'm currently uploading (further discussion at https://github.com/Bumblebee-Project/bumblebee-ppa/pull/34).

**Comment by jethrogb** [comment]
Yes, that is exactly the issue and it has been resolved with the update.
